forms-flow-web/src/components/Modals/Modal.scss

```diff
@@ -0,0 +1,146 @@
+
+.modal-header {
+  justify-content: space-between !important;
+}
+.modal-footer {
+    padding-top: 32px !important;
```

Comment:
Please using spacing css variables wherever px values used for spacing(padding,margin,gap etc..)